accounting/invoice/models.py

```diff
@@ -296,22 +300,29 @@ def upload_to_google_drive(self, invoice_file):  # NOQA
 
         # Generate a query we'll use repeatedly for finding the proper folder.
         query = ("mimeType ='application/vnd.google-apps.folder' and "
-                 "'{}' in parents and "
+                 "'{parent}' in parents and "
                  "trashed=false")
 
-        def get_folder_id(root, name):
-            """ Returns the ID of the folder with name `name` in the folder given by ID `root`. """
-            folders = drive.ListFile({'q': query.format(root)}).GetList()
-            for folder in folders:
-                if folder['title'] == name:
-                    return folder['id']
+        def get_folder_id(root, path=None):
```

Comment:
@UmanShahzad Why use `None` as a default value for `path` (instead of `[]`)?

Reply:
It might be fine in this case, but generally setting a mutable default value [has surprising side effects](http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments).

Reply:
@UmanShahzad Ah, right... Thanks for the reminder! I guess I was mainly looking at this on a conceptual level and forgot to apply appropriate Python-specific filters 😛